src/test/java/com/advanon/pdfsignatures/ValidationTest.java

```diff
@@ -0,0 +1,5 @@
+package com.advanon.pdfsignatures;
+
+class ValidationTest {
```

Comment:
What's the use of this class?

Reply:
Sorry, I forgot to add a TODO:
I left it blank since we don't have CRL and OCSP samples, will update the tests as soon as I have something